plugins/release-dependencies-diff/src/main/kotlin/ReleaseDependenciesDiffFilesTask.kt

```diff
@@ -0,0 +1,102 @@
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+import org.gradle.kotlin.dsl.support.appendReproducibleNewLine
+import java.io.ByteArrayOutputStream
+import java.io.File
+
+abstract class ReleaseDependenciesDiffFilesTask : DefaultTask() {
+
+    @get:Input
+    @set:Option(
+        option = "baselineDependenciesPath",
+        description = "Path to the baseline dependencies directory"
+    )
+    var baselineDependenciesPath: String? = null
+
+    @get:Input
+    @set:Option(
+        option = "dependenciesPath",
+        description = "Path to the dependencies directory"
+    )
+    var dependenciesPath: String? = null
+
+    @get:OutputFile
+    abstract val outputFile: RegularFileProperty
```

Comment:
Do you think it makes sense to generate file only when there are differences? You can mark it optional.
So if we do not have any differences, we won't generate file and won't post its content as a comment to PR as it does not really have any value in this case.

Reply:
we could do this as a future change. unfortunately if someone introduces a change and then reverts it, this wouldn't delete the comment. at least for now this would override the comment